src/ui/uikit/Uikit.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from "react"
+import { Button } from "../Button/Button"
+import { Icon } from "../icon/Icon"
+import { Input } from "../input/input"
+import stl from "./uikit.module.scss"
+
+
+export const Uikit = () => {
+    const [value, setValue] = useState("")
+    return (
+        <div className={stl.uikit}>
+            <div className={stl.df}>
+                <Button>
+                    Вход
+                </Button>
+
+                <Button 
+                    variant="secondary" 
+                    size="sm">
```

Comment:
```suggestion
                    size="sm"
                    >
```

Не, ну так переносы не делаются....

Reply:
исправлено